src/EventProducerService.cpp

```diff
@@ -43,11 +43,12 @@ byte EventProducerService::createDefaultEvent(byte evValue)
   unsigned int nodeNum = module_config->nodeNum;
   
   byte index = module_config->findEventSpace();
-  //eventNum = (int)index + 1;
   for (eventNum = 1; eventNum <= module_config->EE_MAX_EVENTS; eventNum++)
   {
     if (module_config->findExistingEvent(nodeNum, eventNum) == module_config->EE_MAX_EVENTS)
     {
+      controller->sendCMDERR(CMDERR_TOO_MANY_EVENTS);
+      controller->sendGRSP(OPC_RQNPN, getServiceID(), CMDERR_INV_PARAM_IDX);
```

Comment:
Shouldn't the GRSP have the same error code? I.e. CMDERR_TOO_MANY_EVENTS

Reply:
Of course it should!  Don't know how I messed that up.  Now fixed